### ISSUE TITLE
slideToClickedSlide + centeredSlides + loop breaks the loop when sliding to clicked slide

### ISSUE THREAD

**Issue by wagnero11** [nolimits4web/swiper#7478]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://codesandbox.io/p/devbox/swiper-react-infinite-loop-forked-tzdj6h?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522cltycswh000083b6golmfkyzc%2522%252C%2522sizes%2522%253A%255B70%252C30%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522cltycswh000033b6gypfnl3t1%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522cltycswh000053b6gbzrbr4na%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522cltycswh000073b6g0zem57oa%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522cltycswh000033b6gypfnl3t1%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cltycswh000023b6gri695gpt%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252Fsrc%252FApp.jsx%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%255D%252C%2522id%2522%253A%2522cltycswh000033b6gypfnl3t1%2522%252C%2522activeTabId%2522%253A%2522cltycswh000023b6gri695gpt%2522%257D%252C%2522cltycswh000073b6g0zem57oa%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cltycswh000063b6g0m37uf23%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TASK_PORT%2522%252C%2522taskId%2522%253A%2522dev%2522%252C%2522port%2522%253A5173%252C%2522path%2522%253A%2522%252F%2522%257D%255D%252C%2522id%2522%253A%2522cltycswh000073b6g0zem57oa%2522%252C%2522activeTabId%2522%253A%2522cltycswh000063b6g0m37uf23%2522%257D%252C%2522cltycswh000053b6gbzrbr4na%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cltycswh000043b6gpdypfmjd%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522dev%2522%257D%255D%252C%2522id%2522%253A%2522cltycswh000053b6gbzrbr4na%2522%252C%2522activeTabId%2522%253A%2522cltycswh000043b6gpdypfmjd%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D

### Bug description

When clicking on a slide sometimes either the left or right part of the slides glitches or sometimes show blank slides but this cannot be true as loop is set to true, on my local machine left slides work fine but when clicking on right ones only one works and the next after that is empty. It appears if I slide or navigate to the next slide. I'm not sure what seems to be the issue or if I am doing something wrong, I've real several existing closed issues saying to use **loopAdditionalSlides** but I keep getting a message saying Swiper Loop Warning: The number of slides is not enough for loop mode. 

### Expected Behavior

When clicking on a slide it takes me to the clicked slide

### Actual Behavior

When clicking on a slide it breaks the loop or shows empty prev/next slides. Bug better seen if only used slideToClickedSlide and not the navigation arrows or dragging of slides. 

### Swiper version

10.0.0

### Platform/Target and Browser Versions

macOS Chrome V 123.0.6312.124

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

**Comment by ilyasselharak** [comment]
hello yes i got this bug , and like this how you fix it
this because the order of .swiper-slide-prev and swiper-slide-visible and swiper-slide-next inside .slide-wrapper, 
so when using the navigation button next and loop (true) the order will be 
<div class="swiper-wrapper flex">
 <div class="swiper-slide-next">nextSlide</div>
 <div swiper-slide-prev">prevSlide</div>
 <div class="swiper-slide-visible">visibleSlide</div>
</div>
and it should be always in order like the following 
<div class="swiper-wrapper flex">
  <div swiper-slide-prev">prevSlide</div>
   <div class="swiper-slide-visible">visibleSlide</div>
   <div class="swiper-slide-next">nextSlide</div>
</div>

**Comment by wagnero11** [comment]
@ilyasselharak do you mean there is a fix? Sorry, I am confused by your comment

**Comment by ilyasselharak** [comment]
@wagnero11 yes, there's solution i mean when click on next button navigation the order of 3 div whithin the swiper-wrapper got unorder started by .swipe-slide-next and it should be .swipe-slide-prev first then .swipe-slide-visible then .swipe-slide-visible.

the problem only when using navigation with loop,

so some how just render the three divs order .

unorder divs

![image](https://github.com/nolimits4web/swiper/assets/108637546/51eece5c-5ced-45a0-b748-a918f89ed2f1)

and it should be like this order 
order div
![image](https://github.com/nolimits4web/swiper/assets/108637546/3b987657-e3b1-4fac-be5c-4283306efbb2)

**Comment by hipig** [comment]
Add configuration: 
`loopAdditionalSlides=1`

**Comment by adamjrrld** [comment]
Did anyone find a workaround for this? I have the exact same issue.
